v1.1.44 changelog updates (#1067)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-cloud/dbt-cloud-changelog.md

sidebar_label: Changelog
description: "Changelog for the dbt Cloud application"
---
## dbt Cloud v1.1.44 (February 2nd, 2022)

> Along with a fresh look and feel, you can now render a DAG in the IDE seamlessly with a metric node. We've also updated recommended dbt commands in the IDE to include dbt Core v1.0 commands. 

Reviewer: ```suggestion
Along with a fresh look and feel, you can now render a DAG in the IDE seamlessly with a metric node. We've also updated recommended dbt commands in the IDE to include dbt Core v1.0 commands.  

Updates to the scheduler include localized timestamps and how long a run remains blocked, which pinpoints how long a job is unable to run. For example, a run might be blocked waiting for a previous job to finish. Look for these changes in the Timing tab for run details. 
```

sidebar_label: Changelog
description: "Changelog for the dbt Cloud application"
---
## dbt Cloud v1.1.44 (February 2nd, 2022)

> Along with a fresh look and feel, you can now render a DAG in the IDE seamlessly with a metric node. We've also updated recommended dbt commands in the IDE to include dbt Core v1.0 commands.  

Reviewer: ```suggestion
Along with a fresh look and feel, you can now render a DAG in the IDE seamlessly with a metric node. We've also updated recommended dbt commands in the IDE to include dbt Core v1.0 commands, for example, you now see "build" and the "--select" argument.
```

> Along with a fresh look and feel, you can now render a DAG in the IDE seamlessly with a metric node. We've also updated recommended dbt commands in the IDE to include dbt Core v1.0 commands, for example, you now see "build" and the "--select" argument.

Reviewer: ```suggestion
Along with a fresh look and feel, you can now render a DAG in the IDE seamlessly with a metric node. We've also updated recommended dbt commands in the IDE to include dbt Core v1.0 commands, which means you will now see recommendations for "build" and the "--select" argument.
```

Updates to the scheduler include localized timestamps and how long a run remains blocked, which pinpoints how long a job is unable to run. For example, a run might be blocked waiting for a previous job to finish. Look for these changes in the Timing tab for run details. 

#### Performance improvements and enhancements

> * The Command Bar recommendation actions now support dbt Core v1.0 commands. For example, you now see "build" and the "--select" argument.

Reviewer: ```suggestion
* The Command Bar recommendation actions now support dbt Core v1.0 commands. 
```

> * The Command Bar recommendation actions now support dbt Core v1.0 commands. 

Reviewer: ```suggestion
* The Command bar recommendation actions now support dbt Core v1.0 commands. 
```

* Created and Finished times for runs now use localized timestamps.

> * The scheduler now shows blocked time for a run, which pinpoints the time a job is unable to run, such as waiting for a previous job to finish.  

Reviewer: ```suggestion
* The scheduler now shows blocked time for a run.  
```
